### DISCUSSION TITLE
Show Email Banner only if alias did not exist

### DISCUSSION THREAD

**Discussion by thunderrabbit** [anonaddy/anonaddy#167] [Feature Requests]
I have turned email banners off, but I wonder if Email Banner could be shown for emails sent to a new alias.

I see an option to block mail if the alias doesn't exist, so presumably the logic would go there.

**Comment by willbrowningme** [comment]
That should be possible to implement, so you mean only show it if the alias is new and was just created on-the-fly?

Just out of interest what's you main reason for wanting this adding?

**Comment by thunderrabbit** [comment]
I turned off the banner because everything was working smoothly, and I
didn't need to read the banner each time.

However, recently I got a phishing email that looked quite legit.  It had
been sent to mail@(mydomain) and was a perfect replica of the DNS provider
of my domain.

I do not have mail@(mydomain) active, so the mail banner would have alerted
me that this email had come somewhat unexpectedly.
